assets/app/views/images.html

```diff
@@ -10,30 +10,55 @@
+        <h2>
+          {{imageStream.metadata.name}}
+        </h2>
+        <div ng-if="imageStream.spec.dockerImageRepository" class="small">Docker image repository: {{imageStream.spec.dockerImageRepository}}</div>
+        <!-- Print out missing status tags with a warning. If the dockerRepositoryCheck annotation
+             is present, but status tag is missing, tag could not be resolved. Otherwise, it's simply
+             not synced yet. -->
+        <div ng-repeat="specTag in missingStatusTagsByImageStream[imageStream.metadata.name] | orderBy : 'name' track by specTag.name"
+             style="margin-top: 10px;">
+          <div ng-if="imageStream | annotation : 'openshift.io/image.dockerRepositoryCheck'">
+            <span class="pficon-layered" style="margin-right: 5px;"
+                  ng-attr-title="{{imageStream | annotation : 'openshift.io/image.dockerRepositoryCheck'}}">
+              <span class="pficon pficon-warning-triangle"></span>
+              <span class="pficon pficon-warning-exclamation"></span>
             </span>
+            {{specTag.name}}
+            &mdash;
+            <span class="small">Tag could not be resolved.</span>
           </div>
-          <div class="muted small">
-            <div ng-if="imageStream.spec.dockerImageRepository">From Docker repository: {{imageStream.spec.dockerImageRepository}}:{{tag.tag}}</div>
-            <div ng-if="imageStream.spec.tags">
-              <div ng-repeat="specTag in imageStream.spec.tags" ng-if="specTag.name == tag.tag">
-                <div ng-if="specTag.dockerImageReference">From Docker repository: {{specTag.dockerImageReference}}</div>
-                <div ng-if="specTag.from">From image stream: {{specTag.from.namespace}}/{{specTag.from.name}}</div>
-              </div>
+          <div ng-if="!(imageStream | annotation : 'openshift.io/image.dockerRepositoryCheck')">
+              {{specTag.name}}
+              &mdash;
+              <span class="small text-muted">Not yet synced</span>
+          </div>
+        </div>
+        <div ng-repeat="tag in imageStream.status.tags" style="margin-top: 10px;">
+          <div>
+              {{tag.tag}}
+              <span ng-if="tag.items.length && tag.items[0].image">&mdash;
+                <short-id id="{{tag.items[0].image | imageName}}"></short-id>
+                <small class="text-muted">Created <relative-timestamp timestamp="tag.items[0].created"></relative-timestamp></small>
+              </span>
+            <span ng-if="!tag.items.length"><em>No images for this tag</em></span>
+          </div>
+          <div class="small">
+            <!-- dockerImageReference values can get long. Use max width and class truncate to elide some of the SHA1 for long values.
+                 They can still be copied even when elided. -->
+            <div ng-if="tag.items.length && tag.items[0].dockerImageReference" style="max-width: 400px;" class="truncate"
```

Comment:
Here is 300px:

<img width="302" alt="openshift_web_console" src="https://cloud.githubusercontent.com/assets/1167259/8836285/ed1d54b0-308d-11e5-8ed2-1d45cc1e7af6.png">